### PR TITLE
deprecations

### PR DESCRIPTION
**Pull request by raveit65**
Please review.
Note, i noticed an issue with sticky-notes, but it is unrelated to the commit here, happens with master too.
Changing the font color will only apply after a restart of s-n (gtk3).

### CODE REVIEW

**Comment by lukefromdc** [comment]
Tested with GTK 3.21 and seems to work fine here. Panel has menubutton, desktop, launchers, window-list, inhibit, force-quit, window-switcher, hardware sensors, CpuFreq,tray, clock, trash, quit in that order. Tested geyes, fish(not from applets package), weather (not configured), search, invest in another panel, no obvious issues.

File: battstat/battstat_applet.c [hunk 1]
```diff
@@ -1535,13 +1604,24 @@ create_layout(ProgressData *battstat)
...
+#else
   /* Let reconfigure_layout know that the table is currently empty. */
+#endif
   battstat->layout.status = LAYOUT_NONE;
   battstat->layout.text = LAYOUT_NONE;
   battstat->layout.battery = LAYOUT_NONE;
 
+#if GTK_CHECK_VERSION (3, 0, 0)
+  /* Put the grid directly inside the applet and show everything. */
+  gtk_widget_set_halign (battstat->grid, GTK_ALIGN_CENTER);
```
Comment (monsta) [inline battstat/battstat_applet.c]: Hmm is that needed? I don't see the corresponding code for GTK+2...

**Comment by monsta** [comment]
GTK+2 build seems ok.

**Comment by monsta** [comment]
> Changing the font color will only apply after a restart of s-n (gtk3).

Just experimented a bit with both global notes options and properties of the existing note... couldn't reproduce that. Font color and size are changing as they should.

**Comment by monsta** [comment]
Looks ok in GTK+3 as well.

**Comment by raveit65** [comment]
all merged!

**Pull request closed without merging**